### PR TITLE
Adding Dutch and Polish translations [fixes #357 #358]

### LINKED ISSUE #357: Site Translation: Add Polish
### Prize Bounty
0.5 ETH

### Challenge Description
Polish translations are completed, reviewed & ready to to be added to the website (https://github.com/ethereum/ethereum-org-website/issues/325)!

### Submission Requirements
Look at [this previous PR](https://github.com/ethereum/ethereum-org-website/pull/282/files) to see how new languages are added. Steps:
1. Download the [markdown files from CrowdIn](https://crowdin.com/project/ethereumfoundation/pl#):
- https://gist.github.com/samajammin/da816922230762777520edcbcfac513f
- https://gist.github.com/samajammin/c23030fe812535fe85cf858a6e3f304d
- https://gist.github.com/samajammin/ba18c10711d9e3a230961dbb21a63825
- https://gist.github.com/samajammin/b2928961527e5434b911990391a85ea5
- https://gist.github.com/samajammin/8d4cbaa6867df0db284cb438439469bf
2. Add markdown files in a new /docs/pl/ directory
3. Add nav to config.js (`locales` key)
4. Update the homepage markdown file (the root index.md) to use the `HomePage.vue` component
  - Move that content into translations.js (again, it helps to follow [this previous PR](https://github.com/ethereum/ethereum-org-website/pull/282/files))
  - Add `layout: home` in page [front matter](https://vuepress.vuejs.org/guide/frontmatter.html)
5. In each newly added markdown file...
- [Front matter](https://vuepress.vuejs.org/guide/frontmatter.html)
  - Add `meta` property (i.e. `og:title`) - check out other translation markdown files for reference
  - Confirm it has (correct) `lang` property
  - Remove `date` fields (if they exist)
  - Add `sidebar: auto` for /learn & /developers pages
- Review header links & update if needed (e.g. the /use/ page)
  - See [here](https://github.com/ethereum/ethereum-org-website/issues/272) for more context (the header anchor links are dynamic based on the header text, so they change by language)
- Update relative links (i.e. add language path)

### Submission Deadline
The deadline for submissions is November 11, 2019.

### Judging Criteria
The prize will be awarded to the first valid submission.

### Winner Announcement Date
The submission will be reviewed by November 18. Winner announcements and prize payouts will occur before November 31, 2019.

### PR DESCRIPTION
**Pull request by antodp**
Files structure follow the structure in PR #282

### CODE REVIEW

File: docs/.vuepress/theme/utils/translations.js [hunk 1]
```diff
@@ -363,6 +363,96 @@ const translations = {
     'page-learn': '배우기',
     'page-developers': '개발자 가이드'
   },
+  'nl-NL': {
```
Comment (samajammin) [inline docs/.vuepress/theme/utils/translations.js]: I should have specified this (will do for future translations) but please update this key to match language code used by CrowdIn (in this case `nl` - https://crowdin.com/project/ethereumfoundation/nl). This is important for displaying incomplete languages on ethereum.org/languages (you can see we use these keys to compare [here](https://github.com/ethereum/ethereum-org-website/blob/dev/docs/.vuepress/components/LanguagesPage.vue#L53).

File: docs/.vuepress/theme/utils/translations.js [hunk 2]
```diff
@@ -363,6 +363,96 @@ const translations = {
...
+    'page-home-section-enterprise-item-two': '',
+    'page-home-section-enterprise-item-two-link': '',
+    'page-home-section-enterprise-item-three': '',
+    'page-home-section-enterprise-item-three-link': '',    
+    'page-beginners': 'Beginners',
+    'page-use': 'Gebruik',
+    'page-learn': 'Info',
+    'page-developers': 'Ontwikkelaars',
+  },
+  'pl-PL': {
```
Comment (samajammin) [inline docs/.vuepress/theme/utils/translations.js]: Same deal here - please update this key to match CrowdIn (`pl` - https://crowdin.com/project/ethereumfoundation/pl)

File: docs/nl/beginners/index.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
+---
+title: Beginners
+meta:
+  - property: og:title
+    content: Beginners | Ethereum
+lang: nl-NL
```
Comment (samajammin) [inline docs/nl/beginners/index.md]: Same with this `lang` property, you can change to `nl` for consitency (you can likely find & replace all `nl-NL` with `nl`.

File: docs/pl/index.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
+---
+title: Strona główna
+meta:
+  - property: og:title
+    content: Strona główna | Ethereum
+lang: pl-PL
```
Comment (samajammin) [inline docs/pl/index.md]: Same with this `lang` property, you can change to `pl` for consitency (you can likely find & replace all `pl-PL` with `pl`.

File: package-lock.json [hunk 5]
```diff
@@ -0,0 +1,10659 @@
+{
```
Comment (samajammin) [inline package-lock.json]: This file shouldn't have changed. Please remove it.

**Comment by antodp** [comment]
Okay, changes done!
Also I'm planning to make a PR to include RandomList component in translated pages.
Thank you for your attention!

**Comment by samajammin** [comment]
@all-contributors please add @antodp for code

**Pull request merged**